components/benefit_sponsors/app/data_migrations/modify_benefit_application.rb

```diff
@@ -18,7 +18,19 @@ def migrate
   end
 
   def update_aasm_state(benefit_applications)
```

Comment:
you may need to pass more arguments like from_state, to_state and benefit application effective date to make method generic